WebServers/HTTPServer/HttpProcessor.cs

```diff
@@ -340,12 +340,14 @@ public void HandleClient(TcpClient? tcpClient, ushort ListenerPort)
                                                     version = 2;
                                                 else if (absolutepath.Contains("/AirRace/"))
                                                     version = 2;
-                                                else if (absolutepath.Contains("/Uncharted2/"))
+                                                else if (absolutepath.Contains("/uncharted2"))
```

Comment:
The upper-case check needs to remain, apparently a older uncharted 2 scene makes use of it.

Reply:
Okay, I re-add the entry for upper-case